packages/cli/src/tokens/create/index.ts

```diff
@@ -0,0 +1,191 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import type { CssColor } from '@adobe/leonardo-contrast-colors';
+import * as R from 'ramda';
+import { baseColors, generateScaleForColor } from '../../colors/index.js';
+import type { ColorInfo, ColorMode, ThemeColors } from '../../colors/index.js';
+import generateMetadataJson from '../../init/generateMetadataJson.js';
+import generateThemesJson from '../../init/generateThemesJson.js';
+
+type Colors = Record<ThemeColors, CssColor>;
+type Typography = Record<string, string>;
+type TypographyModes = 'primary' | 'secondary';
+
+type CreateTokens = {
+  colors: Colors;
+  typography: Typography;
+  write?: string;
+};
+
+type File = {
+  data: string;
+  path: string;
+  filePath: string;
+};
+
+type DesignTokens = Record<string, { $value: string; $type: string }>;
+type Tokens2ary = Record<string, Record<string, DesignTokens>>;
+type Tokens1ary = Record<string, DesignTokens>;
+type Tokens = Tokens1ary | Tokens2ary;
+
+type Collection = 'theme' | 'global';
+
+const DIRNAME: string = import.meta.dirname || __dirname;
+const DEFAULT_FILES_PATH = path.join(DIRNAME, '../../init/template/default-files/design-tokens/');
+
+const createColorTokens = (colorArray: ColorInfo[]): DesignTokens => {
+  const obj: DesignTokens = {};
+  const $type = 'color';
+  for (let i = 0; i < colorArray.length; i++) {
+    if (i === 13 && colorArray.length >= 14) {
+      obj['contrast-1'] = {
+        $type,
+        $value: colorArray[i].hex,
+      };
+    } else if (i === 14 && colorArray.length >= 15) {
+      obj['contrast-2'] = {
+        $type,
+        $value: colorArray[i].hex,
+      };
+    } else {
+      obj[i + 1] = { $type, $value: colorArray[i].hex };
+    }
+  }
+  return obj;
+};
+
+const generateTypographyTokens = ({ family }: Typography): Tokens => {
+  return {
+    theme: {
+      main: {
+        $type: 'fontFamilies',
+        $value: family,
+      },
+      bold: {
+        $type: 'fontWeights',
+        $value: 'Medium',
```

Comment:
Dette er for vårt theme eller bare nysgerrig hvorfor mappingen? :)
bold => medium
extra-bold => semi bold